test/unit/Types/ScalarTypesResolverTest.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\Types;
+
+use cebe\openapi\spec\Type;
+use OnMoon\OpenApiServerBundle\Types\ScalarTypesResolver;
+use PHPUnit\Framework\Assert;
+use PHPUnit\Framework\TestCase;
+use Safe\DateTime;
+use Safe\Exceptions\DatetimeException;
+
+/**
+ * @covers  \OnMoon\OpenApiServerBundle\Types\ScalarTypesResolver
+ */
+class ScalarTypesResolverTest extends TestCase
+{
+    private ScalarTypesResolver $scalarTypeResolver;
+
+    protected function setUp(): void
+    {
+        $this->scalarTypeResolver = new ScalarTypesResolver();
```

Comment:
Better to deallocate in tearDown, please see
https://tomnewby.net/posts/speed-up-phpunit-1-weird-trick/